nixos/modules/config/fonts/fontconfig.nix

```diff
@@ -65,7 +65,7 @@ let
           ${fcBool cfg.hinting.autohint}
         </edit>
         <edit mode="append" name="hintstyle">
-          <const>hintslight</const>
+          <const>${cfg.hinting.style}</const>
```

Comment:
Setting here to `hintfull` definitively makes a difference, and also the documentation indicates this option still exists.

Reply:
For more:
https://wiki.archlinux.org/title/font_configuration#Hintstyle
https://www.freedesktop.org/software/fontconfig/fontconfig-user.html (search for hintstyle)